File: client-app/params.rb
#!/usr/bin/env ruby

class Params

  def initialize(arg1, arg2, op)
    @arg1 = arg1
    @arg2 = arg2
    @operation = op
  end

  def setArg1(arg)
    @arg1 = arg
  end

  def setArg2(arg)
    @arg2 = arg
  end

  def setOperation(op)
    @operation = op
  end

  def to_json
    {:arg1 => @arg1, :arg2 => @arg2, :operation => @operation}.to_json
  end
end

File: soap-client/params.py
class Params:

    def __init__(self, arg1, arg2, operation):
        self.arg1 = arg1
        self.arg2 = arg2
        self.operation = operation
        pass

    def get_request_body(self):
        return {'arg1': self.arg1, 'arg2': self.arg2, 'operation': self.operation}
